src/labkey/dom/Utils.ts

```diff
@@ -174,4 +208,64 @@ export function signalWebDriverTest(signalName: string, signalResult?: any): voi
     if (signalResult) {
         signalContainer.find('div[name="' + signalName + '"]').attr('value', signalResult);
     }
-}
\ No newline at end of file
+}
+
+/**
+ * Sends a JSON object to the server which turns it into an Excel file
+ * #### Examples
+ *
+ *  ```js
+ *  LABKEY.Utils.convertToExcel(
+ *  {
+ *      fileName: 'output.xls',
+ *      sheets:
+ *      [
+ *          {
+ *              name: 'FirstSheet',
+ *              data:
+ *              [
+ *                  ['Row1Col1', 'Row1Col2'],
+ *                  ['Row2Col1', 'Row2Col2']
+ *             ]
+ *          },
+ *          {
+ *              name: 'SecondSheet',
+ *              data:
+ *              [
+ *                  ['Col1Header', 'Col2Header'],
+ *                  [{value: 1000.5, formatString: '0,000.00'}, {value: '5 Mar 2009 05:14:17', formatString: 'yyyy MMM dd'}],
+ *                  [{value: 2000.6, formatString: '0,000.00'}, {value: '6 Mar 2009 07:17:10', formatString: 'yyyy MMM dd'}]
+ *              ]
+ *          }
+ *     ]
+ *  });
+ *  ```
+ *
+ *  @returns an Excel file to the browser to be saved or opened.
```

Comment:
nit: This function doesn't return anything. The browser returns a file but this function is correctly annotated to return `void`. I'd say just remove this `@returns`.